src/components/ListField/ListField.scss

```diff
@@ -1,54 +1,15 @@
-.react-autosuggest__container {
-  position: relative;
-}
-
-.react-autosuggest__input {
-  width: 100%;
-  height: 30px;
-  padding: 10px 20px;
-  border: 1px solid $gray-600;
-  border-radius: 0.1875rem;
-}
```

Comment:
I don't see this copied over.

Reply:
I've provided explanation in the commit message 89f87af:
```
    refactor: remove unused styles for autosuggest input

    Removing these styles as part of the effor of reducing the use of
    bootstrap variables in SCSS. Were able to remove these styles
    completely, because the `.react-autosuggest__input` class is not applied
    to the input element. Instead this element has class `form-control`,
    because it's overwritten by `inputProps` in
    `src/components/ListField/index.jsx` on lines 183 and 236.

    In other words, these styles are never used, so removing them.
```

Reply:
Added a note to the description about this.